tasks/libs.txt

```diff
@@ -1,12 +1,12 @@
-https://cdn.jsdelivr.net/npm/react@17.0.2/cjs/react.development.js
-https://cdn.jsdelivr.net/npm/moment@2.29.1/moment.js
-https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.js
-https://cdn.jsdelivr.net/npm/vue@2.6.12/dist/vue.js
-https://cdn.jsdelivr.net/npm/lodash@4.17.21/lodash.js
-https://cdn.jsdelivr.net/npm/d3@6.3.1/dist/d3.js
-https://cdn.jsdelivr.net/npm/terser@5.17.4/dist/bundle.min.js
-https://cdn.jsdelivr.net/npm/three@0.124.0/build/three.js
-https://cdn.jsdelivr.net/npm/victory@35.8.4/dist/victory.js
-https://cdn.jsdelivr.net/npm/echarts@5.1.1/dist/echarts.js
-https://cdn.jsdelivr.net/npm/antd@4.16.1/dist/antd.js
-https://cdn.jsdelivr.net/npm/typescript@4.8.3/lib/typescript.js
+https://esm.sh/stable/react@18.2.0/es2022/react.mjs
+https://esm.sh/stable/moment@2.29.4/es2022/moment.bundle.mjs
```

Comment:
We don't benchmark bundled files, they are not representative of real code where we parser and lint.